core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -178,18 +178,19 @@ private FileSystemContext(Subject subject) {
    * Initializes the context. Only called in the factory methods and reset.
    *
    * @param masterInquireClient the client to use for determining the master
-   * @param enableUserMetricsCollection if enables the user metrics collection
+   * @param configuration the instance configuration
    */
   private synchronized void init(MasterInquireClient masterInquireClient,
-      boolean enableUserMetricsCollection) {
+      InstancedConfiguration configuration) {
```

Comment:
use this configuration instead of `Configuration.getMs(PropertyKey.USER_METRICS_HEARTBEAT_INTERVAL_MS)` below?